src/navigation/MainTabNavigator.tsx

```diff
@@ -14,14 +14,14 @@ export const MainTabNavigator = ({
   tabNavigatorConfig: TabNavigatorConfig;
 }) => (
   <Tab.Navigator
-    tabBarOptions={{
-      activeTintColor: tabNavigatorConfig.activeTintColor,
-      inactiveTintColor: tabNavigatorConfig.inactiveTintColor,
-      activeBackgroundColor: tabNavigatorConfig.activeBackgroundColor,
-      inactiveBackgroundColor: tabNavigatorConfig.inactiveBackgroundColor,
-      tabStyle: { marginTop: normalize(0) },
-      style: { backgroundColor: colors.surface },
-      keyboardHidesTabBar: device.platform === 'android'
+    screenOptions={{
+      tabBarActiveTintColor: tabNavigatorConfig.activeTintColor,
+      tabBarInactiveTintColor: tabNavigatorConfig.inactiveTintColor,
+      tabBarActiveBackgroundColor: tabNavigatorConfig.activeBackgroundColor,
+      tabBarInactiveBackgroundColor: tabNavigatorConfig.inactiveBackgroundColor,
+      tabBarStyle: { backgroundColor: colors.surface, marginTop: normalize(0) },
```

Comment:
according to https://reactnavigation.org/docs/upgrading-from-5.x/#the-tabbaroptions-prop-is-removed-in-favor-of-more-flexible-options-for-bottom-tabs the `tabStyle` should have been migrated to `tabBarItemStyle`, right?